### ISSUE TITLE
SMF 3.0: Can't install

### ISSUE THREAD

**Issue by Oldiesmann** [SimpleMachines/SMF#7852]
#### Description
Got a PHP error while trying to set up a test board using the new SMF 3.0 code

```
AH01071: Got error 'PHP message: PHP Fatal error: Uncaught Error: Class "SMF\\Config" not found in /.../Sources/BackwardCompatibility.php:85
Stack trace:
#0 /.../Sources/IntegrationHook.php(340): SMF\\IntegrationHook::exportStatic()
#1 /.../Sources/Autoloader.php(49): require('...')
#2 /.../install.php(56): SMF\\{closure}()
#3 {main}
 thrown in /.../Sources/BackwardCompatibility.php on line 85'
```

### Steps to reproduce
1. Upload files to webserver and move files from "other" as needed
2. Navigate to install.php

### Environment (complete as necessary)
- Version/Git revision: latest SMF 3.0 branch
- Database Type: MariaDB
- Database Version:
- PHP Version: 8.2.12

### Additional information/references

**Comment by jdarwood007** [comment]
Added a fix to my PR #7849
Should get this fixed up

**Comment by Oldiesmann** [comment]
I can confirm that fixed the problem. I just successfully installed SMF 3 on my server (set up a new test site to test it on Postgres as well)
